Numerics/CustomModAdd/CustomModAdd.qs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Numerics.Samples
+{
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Arithmetic;
+    open Microsoft.Quantum.Diagnostics;
+
+    /// # Summary
+    /// Tests a modular addition similar to the one in Fig. 4
+    /// of https://arxiv.org/pdf/quant-ph/9511018v1.pdf
+    ///
+    /// # Input
+    /// ## inputs1
+    /// List of integers to use for the first number
+    /// ## inputs2
+    /// List of integers to use for the second number
+    /// ## modulus
+    /// Modulus used when adding each pair of numbers
+    /// ## numBits
+    /// Number of bits to use to represent each number
+    operation CustomModAdd(inputs1 : Int[], inputs2 : Int[], modulus : Int,
+                            numBits : Int) : Int[]
+    {
+        EqualityFactI(Length(inputs1), Length(inputs2),
+                      "Input arrays need to be of equal length.");
+        mutable results = new Int[Length(inputs1)];
+        for (i in 0..Length(inputs1)-1) {
```

Comment:
```suggestion
        for (i in Arrays.IndexRange(inputs1)) {
```

Reply:
(NB: I suggested adding `Arrays.` here to emphasize where `IndexRange` is coming from. In the context of a sample, that can be a helpful way of clarifying where to find functions and operations.)